### PR TITLE
ADR for accounts and profiles

### PR DESCRIPTION
**Pull request by jenniferplusplus**

### CODE REVIEW

File: docs/decisions/03-accounts-profiles-actors.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+In the longer term, this also creates the possibility that accounts can be connected to more than one profile, and that profiles can be connected to more than one account. This can enable things like easier and more robust management of bot profiles, shared access to profiles representing organizations, or delegating moderation powers between trusted contacts.
+
+## Context
+
+This permits us to separate how people log in from how they would send and receive messages. That's nice for individuals, because it gives them more flexibility to divide up their online persona, if they want to. For instance, having personal and professional profiles. The other thing this enables is to allow a single profile to be controlled by multiple accounts. And I think that is critical to organizations. And that really applies to all organizations.
```
Comment (runewake2) [inline docs/decisions/03-accounts-profiles-actors.md:70]: I'm a really big fan of this separation and I think it enables some really fun future capabilities around OAuth and SSO that I haven't seen elsewhere. Potentially we should consider how multi-account profiles will have responsibilities managed and also consider access restrictions at a per-account level: for example staff that can read and respond privately, but not make public posts, or read-only, etc. Or for example giving a Bot permission to read and post for me, but not manage the profiles settings, delete posts etc.

Also agree with where you are separating these models. Profiles and Accounts is a good slice for this.

**Review by runewake2 — APPROVED** [review]
LGTM, 1 suggestion expanding the scope of this problem a bit around permissions. I think you're already getting at this, but might be handy to call it out explicitly.

File: docs/decisions/03-accounts-profiles-actors.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Discussion
+
+A summary of the comments/issues/concerns that led up to the decision
```
Comment (jenniferplusplus) [inline docs/decisions/03-accounts-profiles-actors.md:76]:
```suggestion
This ADR largely ignores permissions and access controls surrounding shared profiles. That's an important and complex topic, and one that warrants careful consideration. In addition to this account/profile domain modelling, that will also depend on how we implement an authorization model.

[See the PR](https://github.com/Letterbook/Letterbook/pull/37/files)
```

@runewake2 reasonable?

Comment (runewake2) [reply docs/decisions/03-accounts-profiles-actors.md:76]: ✅ Agree, we can revisit when looking into #36

File: docs/decisions/03-accounts-profiles-actors.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Accounts vs Profiles vs Actors
+
+Clarifying the difference between these concepts, and the value of having them be different.
+
+## Status
+
+- [ ] Decided
```
Comment (jenniferplusplus) [inline docs/decisions/03-accounts-profiles-actors.md:7]:
```suggestion
- [x] Decided (2023-07-10)
```

**Pull request merged**